### ISSUE TITLE
Pickle file for the label of vispr dataset is not available.

### ISSUE THREAD

**Issue by wacv2024bidirectional** [UCF-CRCV/TeD-SPAD#5]
I am trying to replicate the results of this paper. However, the pickle file for the vispr dataset's label is not available and I have made the label similar to multiclass classification during the training for the anonymizer model. During the anonymiser model training, after the 30th epoch, I am facing the issue Nan or Inf. I suspect it may be due to the exploding gradient. Have you changed the learning rate during training? The error is:

Epoch 35 started
Train at epoch 35
Learning rate of fa is: 4e-05
Learning rate of fb is: 0.0001
Learning rate of ft is: 0.0001
Training Epoch 35, Batch 0, loss_fa: -5.49380, loss_temporal: nan
Training Epoch 35, Batch 100, loss_fa: nan, loss_temporal: nan
Training Epoch: 35, loss_fa: nan, loss_fb: nan, loss_ft: nan
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
Loaded labels from 8000 files.
Video Validation dataset length: 3783
Video Validation dataset steps per epoch: 472.875
Validation at epoch 35.
Validation Epoch 35, Batch 200, Loss : nan
Epoch 35, mode 0, cropping_fac 0.8 - Accuracy: 1.163%
Running Avg Accuracy for epoch 35, mode 0, cropping_fac 0.8is 1.163%

**Comment by joefioresi718** [comment]
The vispr dataset lablels have been uploaded, and the hyperparameters in the anonymization parameter file have been modified according to our original experiment parameters (they were erroneously set for a failing ablation experiment). This should help avoid this issue.
